resource/source-python/translations/wcs/chat_strings.ini

```diff
@@ -2,277 +2,352 @@
     ru = "{lightgreen}-> Чтобы просмотреть список команд, напишите '{green}wcshelp{lightgreen}' или '{green}wcs{lightgreen}'."
     fi = "{lightgreen}-> Nähdäksesi komento listan, kirjoita '{green}wcshelp{lightgreen}' tai '{green}wcs{lightgreen}'."
+    da = "{lightgreen}-> Skriv '{green}wcshelp{lightgreen}' eller '{green}wcs{lightgreen}' for at se en liste over wcs kommandoer."
 [welcome text]
     en = "{lightgreen}Welcome to this {green}WCS server{lightgreen}. Write '{green}wcshelp{lightgreen}' for help."
     ru = "{lightgreen}Добро пожаловать на этот {green}WCS сервер{lightgreen}. Напишите '{green}wcshelp{lightgreen}' для получения справки."
     fi = "{lightgreen}Tervetuloa tähän {green}WCS palvelimelle{lightgreen}. Kirjoita '{green}wcshelp{lightgreen}' infoon."
+    da = "{lightgreen}Velkommen til denne {green}WCS server{lightgreen}. Skriv '{green}wcshelp{lightgreen}' for at få hjælp."
 [rested xp]
     en = "{lightgreen}You have received {green}{value} rested XP{lightgreen} while you've been offline."
     fi = "{lightgreen}Sinä olet saannut{green}{value} levännyt XP{lightgreen} kun olet ollut poissa paikalta."
+    da = "{lightgreen}Du optjente {green}{value} rested XP{lightgreen} imedens at du var offline."
 [changerace]
     en = "{lightgreen}You've changed your race to: {green}{name}{lightgreen}."
     ru = "{lightgreen}Вы сменили расу на: {green}{name}{lightgreen}."
     fi = "{lightgreen}Sinä vaihdoit sinun racen: {green}{name}{lightgreen}."
+    da = "{lightgreen}Du skiftede din race til: {green}{name}{lightgreen}."
 [changerace warning]
     en = "{lightgreen}You will suicide if changing race while being alive."
     ru = "{lightgreen}Вы сделаете суицид если смените расу будучи живым."
     fi = "{lightgreen}Sinä teet itsemurhan jos vaihdat racen kun olet elossa."
+    da = "{lightgreen}Du vil begå selvmord hvis du skifter race imedens at du er levende."
 [changerace updated]
     en = "{lightgreen}Your race will be changed to {green}{name} {lightgreen}next round."
     ru = "{lightgreen}Ваша раса будет изменена на {green}{name} {lightgreen}в следующем раунде."
     fi = "{lightgreen}Sinun race vaihettaan {green}{name} {lightgreen}seuraavalla erällä."
+    da = "{lightgreen}Din race vil blive skiftet til {green}{name} {lightgreen}i næste runde."
 [gain xp]
     en = "{lightgreen}You have gained {green}{value} XP{lightgreen}."
     ru = "{lightgreen}Вы получили {green}{value} опыта{lightgreen}."
     fi = "{lightgreen}Sinä sait {green}{value} XP{lightgreen}."
+    da = "{lightgreen}Du fik {green}{value} XP{lightgreen}."
 [gain xp killed]
     en = "{lightgreen}You have gained {green}{value} XP{lightgreen} for making a kill."
     ru = "{lightgreen}Вы получили {green}{value} опыта{lightgreen} за убийство."
     fi = "{lightgreen}Sinä sait {green}{value} XP{lightgreen} taposta."
+    da = "{lightgreen}Du fik {green}{value} XP{lightgreen} for at dræbe en modstander.""
```

Comment:
There is "" it only need be " at end of line

Reply:
Thank you srpg, it should now all be fixed and included in the pull request (^.^)